intro-Linux/ch_getting_started.tex

```diff
@@ -3,25 +3,24 @@ \chapter{Getting Started}
 \section{Logging in}
 
 To get started with the HPC-UGent infrastructure, you need to obtain a VSC
-account, see \url{http://hpc.ugent.be/userwiki/index.php/User:VscRequests}.
-
+account, see \href{http://hpcugent.github.io/vsc_user_docs/pdf/intro-HPC-linux-gent.pdf#chapter.2}{Chapter 2 of the HPC manual}.
 \strong{Keep in mind that you must keep your private key to yourself!}
 
-You can look at your public/private key pair as a lock and a key: you give us
+You \OS can look at your public/private key pair as a lock and a key: you give us
```

Comment:
Why is `\OS` included here?! Search & replace gone bad?

Reply:
Yup, I done messed up